### ISSUE TITLE
Export: Invalid header list on overwriting Content-Type header

### ISSUE THREAD

**Issue by codenirvana** [a85/POSTMan-Chrome-Extension#5702]
Overwriting `Content-Type` header added via body mess up exported header list.

1. Create a new request
2. Choose any body type which adds a `Content-Type` header
3. Overwrite the `Content-Type` header
4. Export the collection

```javascript
"header": [
  {
    "key": "Hello",
    "name": "Content-Type",
    "value": "World",
    "type": "text"
  },
  {
    "key": "Content-Type",
    "value": "World"
  }
]
```

 - App Type: Native App
 - Postman Version: 6.6.1
 - OS: macOS Mojave 10.14.2

**Comment by kamalaknn** [comment]
@codenirvana @sujayvenaik Were you able to reproduce this issue?

**Comment by sujayvenaik** [comment]
@kamalaknn Yes, when you edit the key for the content type header and export, it's exporting two headers.

**Comment by neriya197** [comment]
+1

**Comment by vikiCoder** [comment]
This issue is fixed in the latest version of Postman app (v7.23.0).
